### PR TITLE
package deprecation

### PR DESCRIPTION
**Pull request by iamdey**

### CODE REVIEW

**Review by pascalduez — APPROVED** [review]

**Comment by iamdey** [comment]
btw, I have no rights to edit repository description, can you do it @pascalduez  ?

**Comment by pascalduez** [comment]
@iamdey Nope, I can't...

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+**// DEPRECATED: please consider to use [linguijs](https://lingui.js.org/)**
```
Comment (ziir) [inline README.md]: Can we add: `or other alternatives.`

**Review by ziir — CHANGES_REQUESTED** [review]
Can we add a *visible* depreciation warning as a `postinstall` script as well?

**Comment by pascalduez** [comment]
> Can we add a visible depreciation warning as a postinstall script as well?

`npm deprecate @gandi/translate`

**Comment by Pegase745** [comment]
> > Can we add a visible depreciation warning as a postinstall script as well?
> 
> `npm deprecate @gandi/translate`

`npm deprecate @gandi/react-translate` ;)

**Comment by iamdey** [comment]
@ziir I'm not confident with this; please create a new PR. 
I just wanted to act fast.

**Comment by Pegase745** [comment]
@iamdey it has been deprecated, see https://www.npmjs.com/package/@gandi/react-translate

**Comment by yanndinendal** [comment]
:+1:

**Pull request merged**